### PR TITLE
Add elastic-acoustic testcases

### PR DESCRIPTION
**Pull request by krenzland**

### CODE REVIEW

File: convergence_acoustic_elastic/convergence_scholte/mesh_template.geo [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Transfinite Surface {15} = {1,2,3,4};
+Transfinite Surface {17} = {5,6,7,8};
+Transfinite Surface {19} = {1,5,8,4};
+Transfinite Surface {21} = {4,8,7,3};
+Transfinite Surface {23} = {2,6,7,3};
+Transfinite Surface {25} = {1,5,6,2};
+Transfinite Volume{1} = {1,2,3,4,5,6,7,8};
+
+// 6 = periodic, 7 = analytical
+// TODO(Lukas) Uncomment next line
```
Comment (sebwolf-de) [inline convergence_acoustic_elastic/convergence_scholte/mesh_template.geo:64]: Can be removed.

File: convergence_acoustic_elastic/mesh_convergence.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+declare -ri upper_limit=${2}
+
+if [[ ! " ${scenarios[@]} " =~ " ${scenario} " ]]; then
+    echo "$scenario is not a valid scenario! They are: ${scenarios[@]}"
+    exit 1
+fi
+
+echo "Meshing $scenario for levels up to $upper_limit."
+
+mkdir -p meshes
```
Comment (sebwolf-de) [inline convergence_acoustic_elastic/mesh_convergence.sh:22]: I think this statement is not needed.

**Review by sebwolf-de — CHANGES_REQUESTED** [review]
These convergence tests work like the planar wave convergence tests, i.e. one has to parse the SeisSol output for the errors?

**Comment by krenzland** [comment]
> one has to parse the SeisSol output

SeisSol writes a csv file with the errors in the output directory, just like it does for the elastic case.

**Comment by sebwolf-de** [comment]
Ah true, forgot about that feature...

File: convergence_acoustic_elastic/convergence_ocean/parameters.par [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+meshgenerator = 'PUML'        ! Name of meshgenerator (format)
+/
+
+&Discretization
+Order = 6                            ! Order of accuracy in space and times
+Material = 1                         ! Material order
+CFL = 0.5                            ! CFL number (<=1.0)
+FixTimeStep = 5                      ! Manualy chosen minimum time
+ClusteredLts = 2
+/
```
Comment (Thomas-Ulrich) [inline convergence_acoustic_elastic/convergence_ocean/parameters.par:31]: you can remove 
```
Order = 6                            ! Order of accuracy in space and times
Material = 1                         ! Material order
FixTimeStep = 5                      ! Manualy chosen minimum time
iOutputMaskMaterial = 1 1 1          ! Material output
nRecordPoints = 2                    ! number of Record points which are read from file
Interval = 200                       ! Index of printed info at timesteps
printIntervalCriterion = 2           ! Criterion for index of printed info: 1=timesteps,2=time,3=timesteps+time
&Elementwise
printIntervalCriterion = 2
printtimeinterval_sec = 0.01
OutputMask = 1 1 1 1 1 1 1 1 1 1 1
refinement_strategy = 2
refinement = 2
!EndTime = 1.0
/
```

File: convergence_acoustic_elastic/convergence_ocean/rigid.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+!ConstantMap
+    map:
+      map_u_u: -1.0
```
Comment (Thomas-Ulrich) [inline convergence_acoustic_elastic/convergence_ocean/rigid.yaml:3]: add a comment about this parameter.
Why is the file named rigid?

File: convergence_acoustic_elastic/convergence_scholte/parameters.par [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+&Postprocessing
+/
+
+&AbortCriteria
+EndTime = 0.2
+/
+
+&Debugging
+/
```
Comment (Thomas-Ulrich) [inline convergence_acoustic_elastic/convergence_scholte/parameters.par:57]: check comment in the other parameter file.

**Review by Thomas-Ulrich — CHANGES_REQUESTED** [review]
Could you add a README with the reference to these 3 benchmarks?

File: convergence_acoustic_elastic/convergence_ocean/rigid.yaml (hunk 4 above)
Comment (krenzland) [reply convergence_acoustic_elastic/convergence_ocean/rigid.yaml:3]: It's an undocumented feature - it creates a rigid boundary condition (=wall).
This feature will be removed at some point and thus I do not want people to rely on it. 
I'll add a comment in the parameters.par though.

**Review by Thomas-Ulrich — APPROVED** [review]
Awesome!

**Review by sebwolf-de — APPROVED** [review]

**Pull request merged**